vite.config.js

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import path from "path"
+
+export default defineConfig({
+  plugins: [vue({
+    template: {
+      compilerOptions: {
+        isCustomElement: (tag) => tag.includes('ds')
+      }
+    }
+  })],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src')
```

Comment:
Hvis du vil lave unit tests down the road, er det lidt besværligt med aliased imports. 
Så overvej nøje, om det der vindes i developer experience nu og her står mål med tabet af DX i test-øjemed.

Reply:
Giver god mening. Fjerner det for nu.